examples/aws-companion/server.cjs

```diff
@@ -21,14 +26,14 @@ const options = {
       key: process.env.COMPANION_GOOGLE_KEY,
       secret: process.env.COMPANION_GOOGLE_SECRET,
     },
-    s3: {
-      getKey: (req, filename) => `whatever/${Math.random().toString(32).slice(2)}/${filename}`,
-      key: process.env.COMPANION_AWS_KEY,
-      secret: process.env.COMPANION_AWS_SECRET,
-      bucket: process.env.COMPANION_AWS_BUCKET,
-      region: process.env.COMPANION_AWS_REGION,
-      endpoint: process.env.COMPANION_AWS_ENDPOINT,
-    },
+  },
+  s3: {
+    getKey: (req, filename) => `whatever/${Math.random().toString(32).slice(2)}/${filename}`,
```

Comment:
As this is important to get right, showing a real world example (picked from standalone) may be nice

```suggestion
    getKey: (req, filename) => `${crypto.randomUUID()}-${filename}`,
```